app/docker/views/services/edit/includes/configs.html

```diff
@@ -4,7 +4,7 @@
     <rd-widget-body classes="no-padding">
       <div class="form-inline" style="padding: 10px" authorization="DockerServiceUpdate">
         Add a config:
-        <select class="form-control" ng-options="config.Name for config in configs | orderBy: 'Name'" ng-model="newConfig">
+        <select class="form-control !h-[30px] !text-[13px]" ng-options="config.Name for config in configs | orderBy: 'Name'" ng-model="newConfig">
```

Comment:
it could be worth using `<por-select>` here instead of needing to override the styles

Reply:
and in the other places in the PR

Reply:
True.  I might try that.  It's a big form that's had a lot of things wrong.  We could keep going!   I might stop here but we'll see.   It's way better than before.

Reply:
I'd say stop here, merge this PR already, and continue on a separate story :) this pr exists for too long. time to migrate this page to react :laughing: